### ISSUE TITLE
MC 1.8.9 Pump not pulling up lava as expected. 

### ISSUE THREAD

**Issue by Gunner76th** [TeamBR-Modding/NeoTech#265]
FOR 1.9 VERSION: IF YOU HAVE THIS CRASH IT IS FORGE, DON'T POST: 
net.minecraftforge.fml.common.LoaderException: java.lang.NoClassDefFoundError: net/minecraftforge/client/model/TRSRTransformation

What Version of NeoTech are you using:2.4.4

If possible, did you break and replace the block:Yes

Please Pastebin.com the crash Report and put link here:As this is not a crash there is no pastebin. The issue is with the Pump. I have placed it in the nether along with a fluid generator. I primed the generator with a bucket of lava, however the pump is not operating. Below are some photos of the setup, as well as the depth of the pool it is over. 

https://dl.dropboxusercontent.com/u/251115511/Photos/2016-03-30_13.16.35.png
https://dl.dropboxusercontent.com/u/251115511/Photos/2016-03-30_13.16.41.png
https://dl.dropboxusercontent.com/u/251115511/Photos/2016-03-30_13.16.14.png

In addition to the setup shown, I have also tried just supplying the pump with power and placing a tank both beside it, as well as on top it (not all at the same time) and still no lava was placed into the tanks. I even tried removing the pump and leaving the tank thinking it wasnt "seeing" the tank. No matter what I try, I cannot seem to get the pump to pull up lava as shown in the tutorial video.

**Comment by pauljoda** [comment]
Yes, the pump is pretty much broken in 1.9. There are some issues I have to track down and make sure they are actually on our end and not forge. Sadly, I can't offer too much at the moment as we are waiting on forge to stabilize but I will be working on this machine in particular soon

**Comment by Gunner76th** [comment]
This isnt  MC 1.9, this is MC 1.8.9. Sorry guess I should have stated that in the OP.

**Comment by Kriptikz** [comment]
It appears the pump can't pump out water or lava. The pump also only seems to be broken for NeoTech versions 2.4.2, 2.4.3, and 2.4.4. The pump in version 2.4.1 works like a charm. I tested all 4 versions on Forge 1764 and Forge 1808 in order to see if forge was the culprit, and it was not. The pump in version 2.4.1 works with Forge 1808. It seems something was changed from version 2.4.1 to version 2.4.2 that broke the pump. Hope this helps.
